docs/integrations/third-party-libraries.md

```diff
@@ -14,6 +14,7 @@ This can be done using third-party libraries created by the community.
 > This list aims to provide a good starting point
 > for someone looking to extend Zustand's feature set.
 
+
```

Comment:
Probably this line triggers a Prettier error. You can also run prettier so it gets automatically formatted :)

Reply:
Thank you for catching that, sorry I should've double checked all of this.